src/baal/bayesian/weight_drop.py

```diff
@@ -57,8 +30,11 @@ def __init__(self, weight_dropout=0.0, **kwargs):
         wanted = ['in_features', 'out_features']
         kwargs = {k: v for k, v in kwargs.items() if k in wanted}
         super().__init__(**kwargs)
-        weights = ['weight']
-        _weight_drop(self, weights, weight_dropout)
+        self._weight_dropout = weight_dropout
```

Comment:
Where is `weight_drop` now?  i cannot find it :/

Reply:
We do not need it anymore. We call dropout directly onto the weights.